tutorials/dataframe/distrdf001_vecOpsHEP.py

```diff
@@ -0,0 +1,89 @@
+## \file
```

Comment:
`distrdf001_vecOpsHEP` is a "scary" title for the very first tutorial. As per the PPP yesterday, we learned that tutorial number 1 should be a simple hand-holding introduction to the topic (in this case, distRDF).

I think it's ok to use RVecs (we have to assume that users looking at distrdf stuff are familiar with "standard" RDF content) but I would not highlight it so much

Reply:
Ok fair enough, then what's currently called  `distrdf002` represents more of an introduction, e.g. creating a dataset from scratch and plotting a couple of histograms. The only extra step shown there is the connection to a spark cluster which I would label as "more advanced", so I'm a bit in doubt what the introduction could be

Reply:
why would you use distRDF if you don't want to connect to a spark cluster though? also that's basically the only difference w.r.t. "normal" RDF usage, so that's what a distRDF tutorial should highlight -- for explanations/tutorials on existing RDF features you can/should point to existing docs/tutorials

Reply:
Right, the latest commit includes the previously called `distrdf002` tutorial as the only tutorial, which is more verbose in the explanation of the connection to a Spark cluster while also keeping the application very simple